mono/metadata/profiler-events.h

```diff
@@ -101,7 +101,9 @@ MONO_PROFILER_EVENT_1(monitor_failed, MonitorFailed, MonoObject *, object)
 MONO_PROFILER_EVENT_1(monitor_acquired, MonitorAcquired, MonoObject *, object)
 
 MONO_PROFILER_EVENT_1(thread_started, ThreadStarted, uintptr_t, tid)
+MONO_PROFILER_EVENT_1(thread_stopping, ThreadStopped, uintptr_t, tid)
```

Comment:
Does it make sense to have thread_exited introduce a new callback typedef while thread_stopping doesn't?

Reply:
Copy/paste error.